### ISSUE TITLE
Some key combinations are not being picked up correctly

### ISSUE THREAD

**Issue by Stwerp** [ish-app/ish#986]
I mentioned this bug on the discord this weekend, but found that after updating to v1.0.1 the behavior is slightly different. I am writing the bug as of v1.0.1 below and leaving the earlier bug in case that helps track the issue.

I have found problems with several key chords. Example: 

* `Control - Underscore` (`C-_` in emacs parlance) is ignored. This happens with both the software keyboard and an external bluetooth keyboard. 
* `Control - <` and `Control - >` (`C-S-,` and `C-S-.`) are ignored by the software keyboard, and detected as either `,` or `.`, respectively, when using an external bluetooth keyboard.
*  `Meta - <` is detected correctly with the software keyboard, but `Meta - Shift - ,` is detected as `Meta - b` when using an external bluetooth keyboard.

I am running iSH v1.0.1 on an iPad running iOS 14.1.

Issues from a few days ago, prior to v1.0.1:

> I have found problems with several key chords. Example: `Control - Underscore` (C-_ in emacs parlance) is intercepted as `m' when using the software keyboard. When using an external bluetooth hardware keyboard, the key chord is ignored. (In the case of the ext. keyboard, the chord is Control-Shift-Dash keys.)

Possibly related to [this issue.](https://github.com/ish-app/ish/issues/557)

**Comment by Stwerp** [comment]
Just a bit more info: I had not noticed this before, but the terminal.app on Mac OS /also/ tends to ignore the same key combinations. I have not dived deep into the settings, but am guessing this is some lower-level OS functionality that is stealing the key combinations.

**Comment by tbodt** [comment]
Yes, some key combinations are not supported by terminals. C-_ and M-< are possible to support though.

**Comment by JupiLogy** [comment]
Thought I would add that C-/ (undo in emacs) also doesn't get passed through u_u

Old topic I know. Lots of love for this app though :)
